sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientTest.java

```diff
@@ -657,7 +661,7 @@ public void recognizeCustomFormLabeledData(HttpClient httpClient, FormRecognizer
                         .setPollInterval(durationTestMode)).getSyncPoller();
                 syncPoller.waitForCompletion();
                 validateRecognizedResult(syncPoller.getFinalResult(), true, true);
-            }), INVOICE_6_PDF);
+            }), FORM_JPG);
```

Comment:
was this intended? edit: I ask because it looks like we're passing content-type == PDF

Reply:
yes, wanted to use the FORM, updated the type.